docs/index.rst

```diff
@@ -5,15 +5,26 @@
 
 Welcome to PML's documentation!
 ================================
+PML is a python library for accelerator commissioning and beam study. It is a framework that implements high level configure/run behaviour
+of control system components like those used in continuous scans.
+
+PML is a package that includes libraries. Besides the the commonly used applications, like Beam based alignment, the PML includes a set of API
```

Comment:
I misunderstood what beam based alignment meant. I'll remove it from the description.